data/config_files/README.md

```diff
@@ -0,0 +1,85 @@
+# Config files for npg_seq_pipeline
+
+Pipeline-wide parameters can be set here
+
+## Inventory
+- function_list_*.json
+
+  Graph definition files in JSON Graph Format. These define a network of pipeline functions to be performed in dependent order. We have used the metadata key to define a descriptive name for each node as well as any specific resources required by the function. Read more on resource specifications below.
+
+- function_list_*.png
+
+  Graphical renderings of the pipeline graph to show job interdependencies. Node names refer to entries in npg_pipeline::pluggable::registry
+
+- general_values.ini
+
+  Settings for the pipeline in general that constrain global behaviour, e.g. how many Irods errors to tolerate before stopping execution
+
+- wr.json
+
+  A configuration file specifically for the WR scheduler. WR is used in an Openstack environment so it needs to know what VM flavours it can use.
+
+- lsf.ini
+
+  A configuration file specifically for the LSF scheduler used on seqfarm clusters. This file maps generalised queue names to specific queues used by the scheduler. It also has settings for the maximum job array size to schedule as well as token pool settings used to control parallelism of demanding jobs.
+
+## Format of the function_list_* files
+
+    "graph": {
+        "edges": [],
+        "nodes": [],
+        "metadata": {}
+    }
+
+All nodes must be connected via edges. We cannot handle disjoint graphs. Additionally the graph shape should not include cycles and must begin with a "pipeline_start" and end with a "pipeline_end" node.
+
+A node represents a single class of job and the pipeline may create many jobs from a single node. The number of jobs of a given class is defined by the input data.
```

Comment:
Shall we note that, depending on the nature of the data being processed, some nodes can be removed from the network and remaining nodes reconnected( before the jobs are submitted to the scheduler)?